ray-operator/controllers/ray/rayservice_controller.go

```diff
@@ -57,7 +57,7 @@ type RayServiceReconciler struct {
 	Recorder record.EventRecorder
 	// Currently, the Ray dashboard doesn't cache the Serve deployment config.
 	// To avoid reapplying the same config repeatedly, cache the config in this map.
-	ServeConfigs                 cmap.ConcurrentMap[string, string]
+	ServeConfigs                 cmap.ConcurrentMap[string, cmap.ConcurrentMap[string, string]]
```

Comment:
Can you add a comment to explain the definitions of key and value?

Reply:
Added in https://github.com/ray-project/kuberay/pull/2591/commits/ca8482b26c48048a01bbcae4caf61325fa0cdf67